lib/awful/titlebar.lua

```diff
@@ -34,7 +34,8 @@ local all_titlebars = setmetatable({}, { __mode = 'k' })
 -- Get a color for a titlebar, this tests many values from the array and the theme
 local function get_color(name, c, args)
     local suffix = "_normal"
-    if capi.client.focus == c then
+    args.context.has_focus = capi.client.focus == c
```

Comment:
ah, right, I will remove it, I was thinking it would be nicer, but updating the context, umm, context, seem like too much trouble.